### PR TITLE
Enhancement: add YouTube social link

### PR DESCRIPTION
**Pull request by si1k**
#### Description

Added a YouTube Social icon. I matched the YouTube ["required"](https://www.youtube.com/howyoutubeworks/resources/brand-resources/#logos-icons-and-colors) color of #FF0000. 

Included a comment within the site.config.ts file to add clarification if the user does not have a YouTube "custom channel" URL. 

#### Running example

https://alexchaveriat-ie6f7ssgu-si1k.vercel.app/

#### Dark and Light Mode, Side and Bottom, Unselected and Selected
image
image
image
image
image
image
image
image

### CODE REVIEW

File: site.config.ts [hunk 1]
```diff
@@ -20,6 +20,7 @@ export default siteConfig({
   twitter: 'transitive_bs',
   github: 'transitive-bullshit',
   linkedin: 'fisch2',
+  youtube: '#', //use custom channel name or `channel/UCGbXXXXXXXXXXXXXXXXXXXXXX`
```
[the remark this answers is not in the document]
Comment (si1k) [reply site.config.ts]: Good call. Done!

**Comment by si1k** [comment]
Should be good to go. Thank you for the work you do on this repo! Such a cool project! 🤘

**Comment by transitive-bullshit** [comment]
Thanks, Alex — your YouTube channel looks awesome too 💯

**Comment by skybird-dev** [comment]
I've tried adding Instagram using the same technique but had to disable the sidebar because I can't get the Instagram icon to render correctly in the viewport. The color is correct, just won't show the Instagram icon.

**Pull request merged**